ui/pages/confirmations/components/confirm/header/header-info.tsx

```diff
@@ -56,16 +57,35 @@ const HeaderInfo = () => {
 
   const { balance: balanceToUse } = useBalance(fromAddress);
 
+  const isSignatureConfirmation =
+    currentConfirmation &&
+    (currentConfirmation as SignatureRequestType)?.msgParams
+      ?.signatureMethod !== undefined;
+
+  const eventProps = isSignatureConfirmation
+    ? {
+        location: MetaMetricsEventLocation.SignatureConfirmation,
+        signature_type: (currentConfirmation as SignatureRequestType)?.msgParams
+          ?.signatureMethod,
+      }
+    : {
+        location: MetaMetricsEventLocation.Transaction,
+        transaction_type: currentConfirmation?.type,
+      };
+
   function trackAccountModalOpened() {
-    trackEvent({
-      category: MetaMetricsEventCategory.Transactions,
+    const event = {
+      category: isSignatureConfirmation
+        ? MetaMetricsEventCategory.InpageProvider
+        : MetaMetricsEventCategory.Transactions,
```

Comment:
I think we'll want to keep the same category for both Signatures and Transactions for now as  `InpageProvider` event refers to events that are called from the background e.g. `/scripts` directory
```suggestion
      category: MetaMetricsEventCategory.Transactions,
```

would need to update the related tests as well

Reply:
done